readme.md

```diff
@@ -516,6 +516,9 @@ $spaces: (0, 1, 2, 4, 8, 16, auto);
 **Defaults**
 
 ```scss
+/** @type {String} Root font size, used to calculate others font sizes */
+$font-size-base: 16px !default;
```

Comment:
You should add the `$font-size-base` to the "Defintion" section above with a small description of its purpose.